src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private String name = "Isosceles Trapezoid";
```

Comment:
Since the 'name' of the figure does not change, it's better to make it a constant. Also, class-level strings that are constants should be static final and should follow the naming conventions for constants (all uppercase letters with underscores).